lib/readyset/configuration.rb

```diff
@@ -1,39 +1,43 @@
 # lib/readyset/configuration.rb
 require 'active_record'
+require 'yaml'
+require 'erb'
 
 module Readyset
   class Configuration
     attr_accessor :database_url, :shard
 
     def initialize
-      @database_url = ENV['READYSET_URL'] || default_connection_url
+      @database_url = fetch_database_url
```

Comment:
Are we using the database url anywhere in the gem? I think we might be able to remove it altogether

Reply:
Removed! Added spec for shard.